docs/releases/1.1.txt

```diff
@@ -141,7 +141,7 @@ and expected.
 Permanent redirects and the ``redirect_to()`` generic view
 ----------------------------------------------------------
 
-Django 1.1 adds a ``permanent`` argument to the
+Django 1.1 adds a ``permanent`` parameter to the
```

Comment:
Refers to parameter below: https://github.com/lachilles/django/blob/301de774c21d055e9e5a7073e5bffdb52bc71079/django/shortcuts.py#L40-L40